api/placer.py

```diff
@@ -504,5 +504,21 @@ def finalize(self):
             metadata_info = metadata_infos.get(info['name'], {})
             metadata_info.update(info)
             self.metadata['files'].append(metadata_info)
-        self.metadata['_id'] = str(bson.objectid.ObjectId())
         return self.metadata
+
+class AnalysisJobPlacer(AnalysisPlacer):
+    def check(self):
+        super(AnalysisJobPlacer, self).check()
+        self.metadata['outputs'] = self.metadata['acquisition'].pop('files', [])
+
+    def finalize(self):
+        super(AnalysisJobPlacer, self).finalize()
```

Comment:
I inherit from `AnalysisPlacer` mostly to use the `finalize` logic, but also cuts down on some mild code reuse in `process_file_field` and `check`. Currently, neither of them use json schemas for metadata nor REST endpoints. Hoping to change that soon.